scripts/update_teams.py

```diff
@@ -5,13 +5,15 @@
 #  - python
```

Comment:
To be honest we can use `six` or @jjhelmus https://github.com/conda-forge/conda-forge.github.io/pull/136#issuecomment-219113965 suggestion in this PR as it makes the script more general.

I was just making fun of @pelson because he created conda-execute to avoid this sort of problems.